MAINTAINERS.md

```diff
@@ -4,6 +4,10 @@
 | Saurabh Singh          | [getsaurabh02](https://github.com/getsaurabh02)          | Amazon |
 | Subhobrata Dey         | [sbcd90](https://github.com/sbcd90)                      | Amazon |
 | Praveen Sameneni       | [praveensameneni](https://github.com/praveensameneni)    | Amazon |
+| Xuesong Luo            | [xluo-aws](https://github.com/xluo-aws)       | Amazon      |
+| Hailong Cui            | [Hailong-am](https://github.com/Hailong-am)   | Amazon      |
+| Binlong Gao            | [gaobinlong](https://github.com/gaobinlong)   | Amazon      |
+| Zhou Su                | [SuZhou-Joe](https://github.com/SuZhou-Joe)   | Amazon      |
 
 
 
```

Comment:
The format of the MAINTAINERS file should be unified among all repos, see https://github.com/opensearch-project/notifications/blob/main/MAINTAINERS.md, which contains the `Overview` section.

Reply:
Thanks and done